File: atv1.java/src/Calculadora.java
import java.util.Scanner;

public class Calculadora {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite o primeiro número: ");
        double numero1 = scanner.nextDouble();

        System.out.print("Digite o segundo número: ");
        double numero2 = scanner.nextDouble();

        double soma = numero1 + numero2;
        double subtracao = numero1 - numero2;
        double multiplicacao = numero1 * numero2;

        double divisao = (numero2 != 0) ? (numero1 / numero2) : Double.POSITIVE_INFINITY;

        System.out.println("Resultado da Adição: " + soma);
        System.out.println("Resultado da Subtração: " + subtracao);
        System.out.println("Resultado da Multiplicação: " + multiplicacao);
        System.out.println("Resultado da Divisão: " + divisao);

        scanner.close();
    }
}

File: atvC/Calculadora.c
#include <stdio.h>

int main() {
    double numero1, numero2, soma, subtracao, multiplicacao, divisao;

    printf("Digite o primeiro número: ");
    scanf("%lf", &numero1);

    printf("Digite o segundo número: ");
    scanf("%lf", &numero2);

    soma = numero1 + numero2;
    subtracao = numero1 - numero2;
    multiplicacao = numero1 * numero2;

    if (numero2 != 0) {
        divisao = numero1 / numero2;

        printf("Resultado da Adição: %.2lf\n", soma);
        printf("Resultado da Subtração: %.2lf\n", subtracao);
        printf("Resultado da Multiplicação: %.2lf\n", multiplicacao);
        printf("Resultado da Divisão: %.2lf\n", divisao);
    } else {
        printf("Erro: Divisão por zero não permitida.\n");
    }

    return 0;
}
